requirements/base.in

```diff
@@ -16,6 +17,7 @@ django-choices
 django-cors-headers
 django-markup
 django-redis
+git+https://github.com/maykinmedia/django-setup-configuration.git@feature/1-config-command
```

Comment:
I assume this will be changed to a PyPI dependency once the PR is merged and the library is released?

Reply:
You are correct. Git dependency is replaced with PyPI